src/org/usfirst/frc4904/robot/commands/MotorIdleGroup.java

```diff
@@ -0,0 +1,30 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorIdle;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Idles motors of a subsystem.
+ * Used as a default command
+ *
+ */
+public class MotorIdleGroup extends CommandGroup{
+
+	/**
+	 * Run MotorIdle (from WPILib) in parallel on every motor
+	 * from a list of motors with unrestricted size.
+	 *
+	 * @param subsystem
+	 * @param motors
+	 */
+	public MotorIdleGroup(Subsystem subsystem, Motor... motors) {
```

Comment:
This is pretty much what I was asking for, though does it break the previous implementations?

Reply:
it should work